docs/docker/example_commands.md

```diff
@@ -1,32 +1,32 @@
 ### Importing data ###
 
-Use this command to validate a dataset in the folder `./study-dir`, connecting
-to the web API of the container `cbioportal-container`, and import it into the
-database configured in the image, saving an html report of the validation to
-`~/Desktop/report.html`. Note that the paths passed to the `-v` option must be
-absolute paths.
+Use this command to validate a dataset. Specify the study directory by replacing 
+`<path_to_study_directory>` with the absolute path to the study folder. The
+command will connect to the web API of the container `cbioportal-container`, and import 
+the study in its associated database. Make sure to replace `<path_to_report_folder>` with 
+the absolute path were the html report of the validation will be saved.
 
 ```shell
 docker run -it --rm --net cbio-net \
     -v /<path_to_config_file>/portal.properties:/cbioportal/portal.properties:ro \
-    -v "$PWD/study-dir:/study:ro" \
-    -v "$HOME/Desktop:/outdir" \
-    cbioportal-image \
-    metaImport.py -u http://cbioportal-container:8080 -s /study --html=/outdir/report.html
+    -v "<path_to_study_directory>:/study:ro" \
+    -v "<path_to_report_folder>:/report" \
+    cbioportal/cbioportal:latest \
+    metaImport.py -u http://cbioportal-container:8080 -s /study --html=/report/report.html
```

Comment:
Still struggling to import study data, even with the new image.
Attempting to import cbioportal/core/src/test/scripts/test_data/study_es_0 and datahub/public/acbc_mskcc_2015...
Validation step reports: 
> INFO: -: Unable to read xml containing cBioPortal version.
Looking at validateData.py this seems to refer to pom.xml 

Other errors include (study_es_0)...
>ERROR: meta_gsva_pvalues.txt: `geneset_def_version` is different from the geneset_version in the database; value encountered: 'msigdb_6.1'
>ERROR: data_cancer_type.txt: line 1: column 5: Unknown parent for cancer type 'brca-es0'; value encountered: 'Breast'

And for acbc_mskcc_2015...
>ERROR: -: Cancer type of study is neither known to the portal nor defined in a cancer_type file; value encountered: 'acbc'